app/views/admin/ride_uploads/index.html.haml

```diff
@@ -0,0 +1,41 @@
+:css
+  td { padding-right: 15px; }
+  th { padding-right: 25px; }
+  .pending { background-color: red;}
+
+%h3
+  Uploads for #{@ride_zone.name}
+
+%table
+  %thead
+    %tr
+      %th by
+      %th Name
+      %th Description
+      %th Status
+      %th
+
+  %tbody
+    - @ride_uploads.each do |ride_upload|
+      %tr
+        %td
+          = ride_upload.user.name
+        %td
+          = link_to ride_upload.name, admin_ride_zone_ride_upload_path(@ride_zone.slug, ride_upload)
+        %td
+          = ride_upload.description
+          
+        %td{class: ride_upload.status}
+          = ride_upload.status
+          
+        %td
+          = link_to 'Download', rails_blob_path(ride_upload.csv, disposition: "attachment")
```

Comment:
I _think_ it's because I mistakenly submitted the upload form earlier with no CSV selected, but I also get a Rails error on this view now: `signed_id delegated to attachment, but attachment is nil`:

<img width="971" alt="screen shot 2018-10-25 at 9 25 29 pm" src="https://user-images.githubusercontent.com/5596/47540844-91f43a80-d89c-11e8-8f1f-c3b6404a5a4b.png">

Reply:
Yeah, that error related to the new "ActiveStorage" library in Rails, for storing uploads. Must have been a relic of the attachment-less upload, I can't reproduce.